lib/go-parser/go-module.ts

```diff
@@ -13,16 +13,4 @@ export class GoModule {
   public fullName(): string {
     return this.name + "@" + this.version;
   }
-
-  public snykNormalisedVersion(): string {
-    // Versions in Go have leading 'v'
-    let version = this.version.substring(1);
-    // In versions with hash, we only care about hash
-    // v0.0.0-20200905004654-be1d3432aa8f => #be1d3432aa8f
-    version = version.includes("-")
-      ? `#${version.substring(version.lastIndexOf("-") + 1)}`
```

Comment:
are we absolutely sure it's safe to remove this behaviour?

Reply:
Personally, I'm not. But we also have no record of why we have it, and it seems to not work at all.

If you have any ideas on how to figure out if it's safe, let us know ;)

Reply:
Personally, I'm not. But we also have no record of why we have it, and it seems to not work at all.

If you have any ideas on how to figure out if it's safe, let us know ;)

Reply:
I've tagged you in the thread in slack

Reply:
> are we absolutely sure it's safe to remove this behaviour?

Not absolutely sure, but comparing with open source and querying the vulnerability database, it appears `v0.0.0-20210616094352-59db8d763f22` to be more correct than `#59db8d763f22`.